content/en/infrastructure/containers/amazon_elastic_container_explorer.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Amazon Elastic Container (ECS) Explorer
+aliases:
+  - /infrastructure/containers/amazon_elastic_container_explorer
+---
+
+{{< img src="infrastructure/livecontainers/orch_ecs_ex.png" alt="ECS Explorer displaying ECS tasks." style="width:80%;">}}
+
+## Overview
+
+The Datadog Agent and Datadog Amazon ECS integration can retrieve ECS resources for the [ECS Explorer][1]. This feature enables you to monitor the status of EC2 and Fargate tasks, services, and other ECS components across all of your AWS accounts. You can view resource specifications for tasks within a service and correlate them with related logs, metrics, profiling, and more.
+
+### Prerequisites
+
+* **[AWS resource collection][10]**: Required for collecting ECS resources.
+* **[ECS on EC2 integration][2]**: Required for monitoring clusters using the EC2 launch type.
+* **[ECS on Fargate integration][3]**: Required for monitoring clusters using the Fargate launch type.
+* **Agent version >= 7.58.0**: Recommended for a shorter refresh rate on the ECS Explorer page, though it is optional.
```

Comment:
```suggestion
* **Datadog Agent version >= 7.58.0**: Recommended for a shorter refresh rate on the ECS Explorer page.
```

Since we're already saying this is only recommended, it is already understood that it's optional